src/entities/Book/Book.schema.ts

```diff
@@ -0,0 +1,16 @@
+import * as mongoose from 'mongoose';
+import IBook from './Book';
+
+const bookSchema = new mongoose.Schema({
+  id: mongoose.Schema.Types.ObjectId,
+  name: {
+    type: String,
+    required: true,
+    trim: true
+  },
+  ownerId: mongoose.Schema.Types.ObjectId
+});
```

Comment:
Maybe here you could add some additional data that a user may want to add: author of the book, an array of genres, publisher, year of publication (not required but optional)